caluma/core/mutation.py

```diff
@@ -171,13 +174,16 @@ def get_object(cls, root, info, queryset, **input):
 
     @classmethod
     def check_permissions(cls, root, info):
-        for permission_class in settings.PERMISSION_CLASSES:
+        assert (
```

Comment:
`assert` statements may be stripped if user runs it with optimisation, potentially skipping this runtime check. See https://docs.python.org/2.0/ref/assert.html

Reply:
Changed all asserts to ImproperlyConfigured exception. Mind having a look again?